html2text.go

```diff
@@ -3,19 +3,13 @@ package html2text
 import (
 	"bytes"
 	"io"
-	"regexp"
 	"strings"
 	"unicode"
 
 	"golang.org/x/net/html"
 	"golang.org/x/net/html/atom"
 )
 
-var (
-	spacingRe = regexp.MustCompile(`[ \r\n\t]+`)
-	newlineRe = regexp.MustCompile(`\n\n+`)
-)
-
```

Comment:
I would actually not be so radical as to replace the REs at all. They seem to be not too CPU intensive. But as far as the solution works...